digitalocean/README.md

```diff
@@ -0,0 +1,24 @@
+# DigitalOcean
+
+This small setup can be used to provision Flatcar nodes on [DigitalOcean](https://www.digitalocean.com/).
+
+Edit the [Container Linux Config](https://kinvolk.io/docs/flatcar-container-linux/latest/container-linux-config-transpiler/configuration/) `machine-mynode.yaml.tmpl` file if you like, then create the following `terraform.tfvars` with a machine `mynode`, corresponding to the Container Linux Config file name. If you add more machines, create new files for them.
```

Comment:
I find this instructions a bit confusing. If I'm understanding correctly, if I want to have a machine called "marga" then I need to copy machine-mynode.yaml.tmpl and name it machine-marga.yaml.tmpl, and then use "marga" in the machines list, is that correct?

If so, I think I'd word this the other way round:

Create a `terraform.tfvars` file that lists your preferences. Like this one:
```
cluster_name = "mycluster"
machines     = ["mynode"]
datacenter   = "nyc3"
ssh_keys     = ["ssh-rsa AA... me@mail.net"]
```

The machine name listed in the `machines` variable is used to retrieve the corresponding [Container Linux Config](https://kinvolk.io/docs/flatcar-container-linux/latest/container-linux-config-transpiler/configuration/). For each new machine you add to the list, you should copy and modify `machine-mynode.yaml.tmpl` to the corresponding name.

Reply:
Thanks, yes, this order is better to follow. Made a few changes to clarify the relation of the list with the files.